users/templates/user_notifications_list.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+{% load data_structure_element_selectors %}
+
+{% block content %}
+<h1>
+    &nbsp; {% trans 'Notifications' %} &nbsp;
+</h1>
+<br>
+<ul>
+    <div class="container">
+        <div class="table-responsive">
+            {% if users_days_without_report %}
+                <table class="table table-bordered">
+                    <thead>
+                        <tr>
+                            <td>{% trans 'Id' %}</td>
```

Comment:
maybe we could make headers **in bold**?

Reply:
I think we should use CSS for this, it is easier to change it there then.

Reply:
Done